lib/mongoid/extensions/float.rb

```diff
@@ -34,12 +34,11 @@ module ClassMethods
         #
         # @param [ Object ] object The object to mongoize.
         #
-        # @return [ String ] The object mongoized.
+        # @return [ Float | nil ] The object mongoized or nil.
         def mongoize(object)
-          unless object.blank?
-            __numeric__(object).to_f rescue 0.0
-          else
-            nil
+          return if object.nil?
```

Comment:
`return if object.blank?` is fine for all numeric types.

Reply:
It doesn't actually change functionality.... but sure